src/main/java/com/framgia/hepler/ConvertUser.java

```diff
@@ -0,0 +1,39 @@
+package com.framgia.hepler;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Function;
+
+import com.framgia.bean.UserInfo;
+import com.framgia.model.User;
+
+public class ConvertUser {
+	static Function<User, UserInfo> userToUserInfo = (User user) -> {
+		UserInfo info = new UserInfo();
+		info.setId(user.getId());
+		info.setName(user.getName());
+		info.setEmail(user.getEmail());
+		info.setPassword(user.getPassword());
+		info.setPhone(user.getPhone());
+		int role = user.getRole();
+		if (role == 0) {
```

Comment:
`if(user.getRole() == ROLES.USER.ordinal())` sửa thành thế này đi em :p 

https://www.geeksforgeeks.org/enum-in-java/